components/card/package.json

```diff
@@ -23,18 +23,18 @@
     "@spectrum-css/checkbox": "^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0",
     "@spectrum-css/icon": "^3.0.0",
     "@spectrum-css/quickaction": "^3.0.0",
-    "@spectrum-css/typography": "^4.0.0",
-    "@spectrum-css/vars": ">=9.0.0"
+    "@spectrum-css/tokens": "^8.0.1",
+    "@spectrum-css/typography": "^4.0.0"
   },
   "devDependencies": {
     "@spectrum-css/actionbutton": "^3.0.31",
     "@spectrum-css/asset": "^3.0.35",
     "@spectrum-css/checkbox": "^6.0.19",
-    "@spectrum-css/component-builder": "*",
     "@spectrum-css/icon": "^3.0.38",
     "@spectrum-css/quickaction": "^3.0.43",
     "@spectrum-css/typography": "^4.0.31",
-    "@spectrum-css/vars": "^9.0.2",
+    "@spectrum-css/component-builder-simple": "^2.0.6",
+    "@spectrum-css/tokens": "^8.0.1",
```

Comment:
This will likely need to be manually bumped. Thinking out loud but I should add a backlog story to add a utility to handle version bumps in WIP branches when things change during review.